airgun/exceptions.py

```diff
@@ -1,4 +1,6 @@
 """Exceptions raised by airgun"""
+from selenium.common.exceptions import InvalidElementStateException
+from widgetastic.exceptions import *  # noqa: F403, F401
```

Comment:
AFAIK we have imported the exceptions here from widgetastic but didnt add it into the `__all__` list so that all those exceptions could be accessed from airgun.

I think this also needs to be fixed in airgun master!

Reply:
Airgun modules can continue to import from widgetastic.exceptions directly, as its a direct dependency of the library.  

Can you be more specific about what specifically needs to be fixed here? This commit diff matches master.

Reply:
Non blocking comment, but still want to hear if we've got some issue here. :)